docker/ubuntu-based/prod/vswitch/vppctl

```diff
@@ -4,6 +4,14 @@
 
 set -euo pipefail
 
+get_vswitch_container_ID() {
```

Comment:
Having ID caps at the end looks weird, it's not really consistent.  I'd prefer lower case.